tests/integration/cli/test_check_docs_cli.py

```
@@ -0,0 +1,42 @@
+
+import pytest
+
+PROJECT_ROOT = Path(__file__).resolve().parents[3]
+
+
+@pytest.mark.integration
+def test_check_docs_cli_passes(tmp_path: Path) -> None:
+    """
+    Run the check_docs CLI against the project root and assert it reports a passing status.
+    
+    Executes the scripts.check_docs module with --strict and writes JSON output to a temporary file; fails the test if the process exits non‑zero and asserts that the JSON payload's "status" field equals "pass".
```

Ruff (0.13.2)

Docstring contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)